### PR TITLE
2019-10-29 |  Django Custom Commands & Django Admin

### PR DESCRIPTION
**Pull request by iamhaiderkhan**
- Custom Commands
  -- Create Users
  -- Current Time
  -- DateTimeStamps

- Django Admin with Custom Users fields.
 -- Filters
 -- Searchable Fields

### CODE REVIEW

File: date_times_stamps/admin.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```
Comment (ziafazal) [inline date_times_stamps/admin.py]: what is the usage of this admin file?

File: users/forms.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+from django.contrib.auth.forms import UserCreationForm
+from .models import CustomUser
+
+
+class CustomUserForm(UserCreationForm):
```
Comment (ziafazal) [inline users/forms.py]: Why you need this form definition?

**Review by ziafazal — CHANGES_REQUESTED** [review]
@iamhaiderkhan I don't we need timestamps app now.

File: users/forms.py (hunk 2 above)
Comment (iamhaiderkhan) [reply users/forms.py]: I have used this form in Custom User Admin Modal for custom fields while adding a new user via Django admin.

Comment (ziafazal) [reply users/forms.py]: You don't need this `CustomUserForm` since all you are doing is declaring fields of the model in it.

**Review by ziafazal — APPROVED** [review]
👍

**Pull request merged**